wdl/EvidenceQC.wdl

```diff
@@ -169,5 +194,66 @@ workflow EvidenceQC {
     File bincov_matrix = MakeBincovMatrix.merged_bincov
     File bincov_matrix_index = MakeBincovMatrix.merged_bincov_idx
     File bincov_median = MedianCov.medianCov
+
+    File qc_table_csv = MakeQcTable.qc_table_csv
   }
 }
+
+task MakeQcTable {
+  input {
+    String? prefix
+
+    File? ploidy_plots
+    File bincov_median
+    File WGD_scores
+    Float? melt_insert_size
+
+    File? manta_qc_low
+    File? manta_qc_high
+    File? melt_qc_low
+    File? melt_qc_high
+    File? wham_qc_low
+    File? wham_qc_high
+    File? scramble_qc_low
+    File? scramble_qc_high
+
+    String sv_pipeline_docker
+    RuntimeAttr? runtime_override
+  }
+
+  output {
+    File qc_table_csv = "${prefix}_table.csv"
+  }
+
+  command <<<
+    set -euxo pipefail
+
+    PLOIDY=""
+    if ~{defined(ploidy_plots)}; then
+      PLOIDY="--ploidy-plots ~{ploidy_plots}"
+
+    python sv-pipeline/scripts/make_evidence_qc_table.py \
+      $PLOIDY
```

Comment:
You can do the argument setup for optional inputs in one line with `~{"--ploidy-plots " + ploidy_plots}`